zebrad/tests/common/get_block_template_rpcs/get_block_template.rs

```diff
@@ -27,9 +30,18 @@ use crate::common::{
 /// We've seen it take anywhere from 1-45 seconds for the mempool to have some transactions in it.
 pub const EXPECTED_MEMPOOL_TRANSACTION_TIME: Duration = Duration::from_secs(45);
 
+/// Delay between getting block proposal results and cancelling long poll requests.
+pub const EXTRA_LONGPOLL_WAIT_TIME: Duration = Duration::from_millis(150);
+
+/// Delay between attempts to validate a template as block proposals.
+///
+/// Running many iterations in short intervals tests that long poll requests correctly
+/// return `submit_old: false` when the old template becomes invalid.
+pub const BLOCK_PROPOSAL_INTERVAL: Duration = Duration::from_millis(300);
+
 /// Number of times we want to try submitting a block template as a block proposal at an interval
 /// that allows testing the varying mempool contents.
-const NUM_SUCCESSFUL_BLOCK_PROPOSALS_REQUIRED: usize = 10;
+const NUM_SUCCESSFUL_BLOCK_PROPOSALS_REQUIRED: usize = 1000;
```

Comment:
This seems like an unusually large number of test iterations to commit to CI, how long does it take?
We typically use 256 iterations for quick proptests, and 1-32 for slower tests.

Optional: If we need a larger number for some reason, please document it here. (Or refer to `BLOCK_PROPOSAL_INTERVAL` if the reason is covered by its docs - it seems like it might be, but it's easy to miss.)

Reply:
> This seems like an unusually large number of test iterations to commit to CI, how long does it take?

It takes ~15m locally for me, I think 6-30 minutes in general. 

It waits 45s for the mempool, and after that, `try_validate_block_template` seems to take about ~900ms, so the 300ms interval is now in parallel to those requests is effectively ignored right now. I think I should change that to 1 second.

Reply:
Ok, I think that's reasonable, but can you please check that this is not the longest CI job now?

If it is, we could reduce the number of iterations to 500.